src/pyret-ide.js

```diff
@@ -26,12 +27,14 @@ export default {
    * @param {boolean} [config.skipCSSLoading=false] - Whether or not to skip loading
    *                                                  of CSS used by pyret-ide
    */
-  init({rootEl, runtimeApiLoader, debug, skipCSSLoading}) {
+  init({rootEl, runtimeApiLoader, debug, skipCSSLoading, firebaseConfig}) {
     if (!skipCSSLoading) {
       require('bootstrap/less/bootstrap.less');
     }
     const store = createStore({debug});
     store.dispatch(loadRuntimeApi(runtimeApiLoader));
+    console.log("initializing firebase with", firebaseConfig);
+    firebase.initializeApp(firebaseConfig);
```

Comment:
if no `firebaseConfig` gets passed in, don't try to initialize. Instead log a warning message with `console.warn`